site/content/blog/2023-09-09-browser-extension/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Parsing .DS_Store files with ZAP"
+summary: "ZAP Spider can now probe and parse macOS' .DS_Store files."
```

Comment:
Needs update.

Reply:
"GSoC 2023 Browser Recorder" (or similar) might be a better blog title than "Browser Recorder - Exploring My Google Summer of Code 2023 Journey" - its better to keep them shorter (IMO). You can still use that title lower down..